### PR TITLE
SMS guide for a delivery feedback

### PR DESCRIPTION
**Pull request by miksta-sinch**

### CODE REVIEW

File: docs/sms/sms-guide.md [hunk 1]
```diff
@@ -722,6 +723,65 @@ The response is a JSON object with the following fields:
 
 The batch ID is not known to the system or the recipient is not a target of this batch.
 
+
+### Send a delivery feedback
+
+Send a feedback if your system can confirm successfully message delivery.
```
Comment (thomasf147) [inline docs/sms/sms-guide.md:729]: successful message delivery

Comment (miksta-sinch) [reply docs/sms/sms-guide.md]: Thank you! fixed

File: docs/sms/sms-guide.md [hunk 2]
```diff
@@ -159,6 +159,7 @@ JSON body fields:
...
 |client_reference                   |The client identifier of batch message. If set, it will be added in the delivery report/callback of this batch     |      String      |N/A                 |                            Max 128 characters long                            |                      No                      |
 |max_number_of_message_parts        |Message will be dispatched only if it is not split to more parts than Max Number of Message Parts                  |      String      |N/A                 |                           Must be higher or equal 1                           |                      No                      |
+|feedback_enabled                   |If set to true then [feedback](#send-a-delivery-feedback) is expected after successfull delivery.                  |      Boolean     |false               |                            true or false                                      |                      No                      |
```
Comment (thomasf147) [inline docs/sms/sms-guide.md:162]: successful

Comment (miksta-sinch) [reply docs/sms/sms-guide.md]: successfully fixed

File: docs/sms/sms-guide.md [hunk 3]
```diff
@@ -722,6 +723,67 @@ The response is a JSON object with the following fields:
...
+
+
+`200 OK`
+
+The response is a JSON object with the following fields:
+
+|Name                |Description                                             |JSON Type        |
+|--------------------|--------------------------------------------------------|:---------------:|
+|recipients          |List of MSISDNs that successfully received the message  |   String array  |
+
```
Comment (mmcgeurts) [inline docs/sms/sms-guide.md:777]: Is the response a list of all MSISDNs referenced in this request? or if a second "delivery_feedback" event comes in, the full list is listed (the first feedback event + the second)?

Comment (miksta-sinch) [reply docs/sms/sms-guide.md:777]: Good question!
Currently, the response has only MSISDNs from the request, so there is no value in this actually. We can:

1. Respond with 202 ACCEPTED with an empty body in the response.
2. Respond with MSISDNs from all events as you suggested. 
3. Keep it as it is currently but improve description.

Comment (miksta-sinch) [reply docs/sms/sms-guide.md:763]: Changed response to 202 Accepted with an empty body. Please confirm if that is ok for you.

**Review by thomasf147 — APPROVED** [review]

**Comment by spacedsweden** [comment]
Hey
Are you waiting for someone to approve or are we good to merge?

**Comment by thomasf147** [comment]
This is feature is not generally available yet so it will have to wait to be merged

**Pull request merged**